actionpack/test/controller/mime/accept_format_test.rb

```diff
@@ -86,8 +86,9 @@ def test_format_with_inherited_layouts
     assert_equal '<html><div id="super_iphone">Super iPhone</div></html>', @response.body
   end
 
-  def test_non_navigational_format_with_no_template_fallbacks_to_html_template_with_no_layout
-    get :index, format: :js
-    assert_equal "Hello Firefox", @response.body
+  def test_non_navigational_format_with_no_template_raises
```

Comment:
It's not really clear what "non-navigational" means here, and I think we should document _why_ it raises.

```suggestion
  def test_js_format_with_no_template_does_not_fall_back_to_html_template
```

(Sorry I keep nitpicking test names!)

Reply:
> (Sorry I keep nitpicking test names!)

No problem, it's great!